### ISSUE TITLE
Ipatool not working.

### ISSUE THREAD

**Issue by plyght** [majd/ipatool#291]
### What happened?

I attempted to purchase an application using ipatool, but the purchase failed with the error message as seen below.
<img width="846" alt="image" src="https://github.com/user-attachments/assets/35fe2a1d-533d-4937-8489-2f422d162c17">

I have aswell made several attempts on different systems, including my Mac running Sequoia Dev Beta 8 and my Raspberry Pi 5B. 

I have also ensured I am signed in to my correct account, as seen below.
<img width="846" alt="image" src="https://github.com/user-attachments/assets/30f25861-86f9-4523-9b01-bca6fbc4c07f">

### Version

2.1.4

### Relevant log output

```shell
ipatool purchase -b com.bsakhuja.Epicentral --verbose
1:00PM DBG error="failed to purchase item with param 'STDQ': failed to purchase app" metadata={"Data":{"CustomerMessage":"Unable to process your request.","FailureType":"","JingleDocType":"","Status":0},"Headers":{"Apple-Originating-System":"MZBuy","Apple-Seq":"0","Apple-Timing-App":"50 ms","Apple-Tk":"false","B3":"d2380a4c1aa205d9986ac99df4fef7ca-c22b14ecc0795bd3","Cache-Control":"private, no-cache, no-store, no-transform, must-revalidate, max-age=0","Connection":"keep-alive","Content-Type":"text/xml","Date":"Sun, 08 Sep 2024 17:00:08 GMT","Edge-Control":"no-store; cache-maxage=0","Expires":"Sun, 08 Sep 2024 17:00:08 GMT","Itspod":"70","Pod":"70","Server":"Apple","Set-Cookie":"countryVerified=1;Version=1;Path=/WebObjects;Domain=.apple.com; itspod=70;Version=1;Path=/;Domain=.apple.com;Expires=Tue, 08-Oct-2024 17:00:07 GMT;Max-Age=2591999; wosid-lite=\"\";Version=1;Path=/WebObjects;Expires=Thu, 01-Jan-1970 00:00:00 GMT;Max-Age=0;Secure; wosid-lite=\"\";Version=1;Path=/WebObjects;Domain=.apple.com;Expires=Thu, 01-Jan-1970 00:00:00 GMT;Max-Age=0;Secure; wosid-lite=Mba6JdyLDQt0lh5VgtPbK0;Version=1;Path=/;Domain=.apple.com;Secure","Strict-Transport-Security":"max-age=31536000; includeSubdomains","X-Apple-Date-Generated":"Sun, 08 Sep 2024 17:00:08 GMT","X-Apple-Jingle-Correlation-Key":"2I4AUTA2UIC5TGDKZGO7J7XXZI","X-Apple-Lokamai-No-Cache":"true","X-Apple-Request-Store-Front":"143441-1,34","X-Apple-Request-Uuid":"d2380a4c-1aa2-05d9-986a-c99df4fef7ca","X-Apple-Translated-Wo-Url":"/WebObjects/MZBuy.woa/wa/buyProduct","X-B3-Spanid":"c22b14ecc0795bd3","X-B3-Traceid":"d2380a4c1aa205d9986ac99df4fef7ca","X-Content-Type-Options":"nosniff","X-Frame-Options":"SAMEORIGIN","X-Responding-Instance":"MZBuy:3700148:::","X-Xss-Protection":"1; mode=block"},"StatusCode":200}
1:00PM ERR error="failed to purchase item with param 'STDQ': failed to purchase app" success=false
```

**Comment by Sicksyg** [comment]
Duplicate of #284

**Comment by pradoAgustin** [comment]
Everyone is with the same bug, are any updates on this ? we are cracking our heads with the possible solutions but nothing is working

**Comment by iteroji** [comment]
> Everyone is with the same bug, are any updates on this ? we are cracking our heads with the possible solutions but nothing is working

The endpoint on Apple's side has been disabled, and obtaining a license is not possible. Downloading still works once the app is purchased on an iPhone or mac. Currently there are no known solutions to this.

**Comment by majd** [comment]
Closed as duplicate of #284.
